### PR TITLE
Leetcode Problem 542

### PR DESCRIPTION
**Pull request by PraSarOG**

### CODE REVIEW

File: 542.01-matrix.cpp [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+ * @lc app=leetcode id=542 lang=cpp
+ *
+ * [542] 01 Matrix
+ */
+
+// @lc code=start
```
Comment (cheemsjs) [inline 542.01-matrix.cpp]: You can remove this comment

File: 542.01-matrix.cpp [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                    vis[row][col]=1;
+                }                
+            }
+        }
+
+        return dis;
+
+    }
+};
+// @lc code=end
```
Comment (cheemsjs) [inline 542.01-matrix.cpp]: also, this comment is not needed

**Review by cheemsjs — CHANGES_REQUESTED** [review]
I have added my review for your code, please resolve

File: 542.01-matrix.cpp (hunk 2 above)
Comment (PraSarOG) [reply 542.01-matrix.cpp]: okay

**Comment by cheemsjs** [comment]
This solution seems invalid

**Pull request closed without merging**